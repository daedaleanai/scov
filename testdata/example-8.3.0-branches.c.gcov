version:8.3.0
file:example.c
function:28,59,1,main
lcount:28,1,0
lcount:34,1,0
branch:34,taken
branch:34,nottaken
lcount:36,1,0
lcount:37,1,0
lcount:43,1,0
lcount:44,1,0
lcount:49,1,0
branch:49,nottaken
branch:49,taken
lcount:51,0,1
lcount:55,1,0
lcount:58,1,0
