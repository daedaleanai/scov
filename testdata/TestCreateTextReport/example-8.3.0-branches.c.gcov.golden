 Lines	 Funcs	Branch	Region
------	------	------	------
  90.0%	100.0%	 50.0%	   --%	example.c
------	------	------	------
 90.0%	100.0%	 50.0%	   --%	Overall
